### PR TITLE
Update "Cursor Visibility and Catching" page for libGDX 1.11.0

### PR DESCRIPTION
**Pull request by Frosty-J**
This is written for the upcoming version of libGDX, though the only difference it makes is 4 new system cursors. If you want to avoid confusion, that part can be left out for the time being.

The current live version of this page has broken images due to hotlinking mozilla.org.

### CODE REVIEW

File: wiki/input/cursor-visibility-and-catching.md [hunk 1]
```diff
@@ -1,40 +1,138 @@
...
+
+## Additional resources
+
+If you wish to let your HTML5 game use system cursors libGDX doesn't support, this is a good starting point:
+
+* [CSS `cursor` values](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#values)
+* [Interfacing with platform specific code](https://libgdx.com/wiki/app/interfacing-with-platform-specific-code)
+* [JSNI](http://www.gwtproject.org/doc/latest/DevGuideCodingBasicsJSNI.html)
+
+If you're willing to modify libGDX's source code, it is possible to properly hide the mouse cursor:
```
Comment (crykn) [inline wiki/input/cursor-visibility-and-catching.md:133]: Would you mind changing that to reflect https://github.com/libgdx/libgdx/pull/6822?

Comment (Frosty-J) [reply wiki/input/cursor-visibility-and-catching.md:133]: Working on it. I've just noticed an issue with the cursors that I'm getting to the bottom of. If https://github.com/libgdx/libgdx/pull/6821 is merged I can also update it to reflect that, though at no comments I'm not holding my breath.

Comment (Frosty-J) [reply wiki/input/cursor-visibility-and-catching.md:133]: Hm, so `move` is the same as `grabbing` in Firefox on my system. Fine on Chrome and fine on Windows. Probably something to do with https://github.com/ubuntu/yaru/issues/2880 so I won't note it.

Comment (crykn) [reply wiki/input/cursor-visibility-and-catching.md]: > If https://github.com/libgdx/libgdx/pull/6821 is merged I can also update it to reflect that, though at no comments I'm not holding my breath.

Android isn't really my jam, so we'll have to wait for someone else to review that.

**Pull request merged**